### ISSUE TITLE
Unable to scan using SCAP WORKBENCH for Windows 7

### ISSUE THREAD

**Issue by kvashishta** [OpenSCAP/scap-workbench#141]
I am having issues connecting from my windows 7 workstation with scap-workbench 1.1.5 to a CentOS 7.3 system, while I can ssh as root to the CentOS machine, I cannot scan using scap workbench, scap workbench seems to be wanting to create a directory on the remote system (CentOS 7)using my  Active Directory credentials instead of "root", I login to my workstation using the AD credentials and have uninstalled and reinstalled SCAP WORKBENCH......it  tries to create a directory called /home/<AD username>/.ssh
https://pastebin.com/QPcY5FrG
scanning with IP:
https://pastebin.com/FM61yryC

**Comment by kvashishta** [comment]
That is /home/<AD username> /.ssh  not //.ssh

**Comment by dahaic** [comment]
@kvashishta can you past here output of the "dry run" option? i.e. set workbench the same way you observe the issue, and check `dry run` tick next to Scan. It could shed some light into the issue.

**Comment by kvashishta** [comment]
oscap-ssh root@<server name> 22 xccdf eval --datastream-id scap_org.open-scap_datastream_from_xccdf_ssg-rhel7-xccdf-1.2.xml --xccdf-id scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml --profile xccdf_org.ssgproject.content_profile_ospp-rhel7 --oval-results --results /tmp/xccdf-results.xml --results-arf /tmp/arf.xml --report /tmp/report.html "C:/Program Files (x86)/scap-workbench/ssg/ssg-centos7-ds.xml"

**Comment by jan-cerny** [comment]
@WesleyCeraso Have you something about this?

**Comment by matejak** [comment]
@kvashishta I have noticed two things:
* SSH under Windows may not always work due to the fact that `Cygwin` implementation of it is used and some very low-level errors may occur. Restarting the computer may help, for full list of what can go wrong see https://unix.stackexchange.com/questions/55909/cygwin-readline-lib-loaded-in-different-locations
* Maybe the hostname substitution is not working as expected? I am judging from `oscap-ssh root@ 22 xccdf eval --datastream-id ...` (where is the hostname past `@`?). Did you remove it and did you mask the IP address/hostname in pastebin logs?

**Comment by kvashishta** [comment]
Yes I removed the hostname I tried with both hostname and IP address, I
also restarted the workstation
